server/src/controllers/note.controllers.ts

```diff
@@ -2,8 +2,21 @@ import { Request, Response } from 'express';
 import { Note } from '../models/note.model';
 
 export const createNote = async (req: Request, res: Response) => {
-  res.send('creating a note.' + JSON.stringify(req.body));
-  console.log(req.body.content);
+  const { author, title, content, creationDate } = req.body;
+
+  try {
+    if (!(author && title && creationDate)) {
+      // we allow creating a note without providing the content initially
+      res.status(404).json({
+        error: 'Specify author, title and creation date',
```

Comment:
I think we can make the error message even more precise by specifying exactly what is missing.